frontend/src/components/IssueForm/styles.js

```diff
@@ -0,0 +1,35 @@
+import {StyleSheet} from 'aphrodite';
+
+const styles = StyleSheet.create({
+  wrapper: {
+    display: "flex",
+    flexDirection: "column",
+    alignItems: "center"
+  },
+  image_title: {
+    margin: "1em auto",
+    fontSize: "1.5em"
+  },
+  images_wrapper: {
+    margin: "1em auto"
+  },
+  upload_wrapper: {
+    flexDirection: "row"
+  },
+  form: {
+    maxWidth: "600px",
+    display: "grid",
+    gridTemplateColumns: "100px 1fr",
+    gridGap: "8px"
+  },
+  left_grid: {
+    lineHeight: "1.5em",
+    textAlign: "right",
+    gridColumn: "1 / 2"
+  },
+  right_grid: {
+    gridColumn: "2/3"
+  }
+});
+
+export default styles;
```

Comment:
I mean i've already talked about my disagreement about putting into two separate files. But if you keep it like this you might aswell use Css. Plus if you export default just do it on one line. `export default const styles` There's not point in separating/I don't see the point.

Reply:
sorry I didn't think we had come to a decision and in this case the styles could go in the same file. Let's discuss this next chat. Happy to change it if I'm outvoted :)